hoomd/md/pair/pair.py

```diff
@@ -452,6 +452,94 @@ def __init__(self, nlist, default_r_cut=None, default_r_on=0., mode='none'):
         # correctly?
 
 
+class ExpandedLJ(Pair):
+    r"""Expanded Lennard-Jones pair potential.
+
+    Args:
+        nlist (`hoomd.md.nlist.NList`): Neighbor list.
+        default_r_cut (float): Default cutoff radius :math:`[\mathrm{length}]`.
+        default_r_on (float): Default turn-on radius :math:`[\mathrm{length}]`.
+        mode (str): Energy shifting mode.
+
+    `ExpandedLJ` specifies that a radially shifted Lennard-Jones
+    type pair potential should be applied between every non-excluded
+    particle pair in the simulation.
+
+    .. math::
+        :nowrap:
+
+        \begin{eqnarray*}
+        V_{\mathrm{ExpandedLJ}}(r)  = & 4 \varepsilon \left[ \left(
+                                \frac{\sigma}{r - \Delta} \right)^{12} -
+                                \left( \frac{\sigma}{r - \Delta}
+                                \right)^{6} \right]; & r < (r_{\mathrm{cut}}) \\
+                             = & 0; & r \ge (r_{\mathrm{cut}}) \\
+        \end{eqnarray*}
+
+    See `Pair` for details on how forces are calculated and the
+    available energy shifting and smoothing modes.
+
+    Attention:
+        Due to the way that `ExpandedLJ` modifies the cutoff criteria, a smoothing
+        mode of *xplor* is not supported.
```

Comment:
`ExpandedLJ` no longer modifies the cutoff. `xplor` will smooth the potential correctly.